apps/spruce/src/pages/version/TestAnalysis/GroupedTestMapList/FailedTestGroup/index.tsx

```diff
@@ -0,0 +1,96 @@
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import TaskStatusBadge from "@evg-ui/lib/components/Badge/TaskStatusBadge";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { TaskStatus } from "@evg-ui/lib/types/task";
+import { Accordion } from "components/Accordion";
+import { trimStringFromMiddle } from "utils/string";
+import { TaskBuildVariantField } from "../../types";
+import FailedTestGroupTable from "./FailedTestGroupTable";
+
+interface FailedTestGroupProps {
+  testName: string;
+  tasks: TaskBuildVariantField[];
+}
+
+const FailedTestGroup: React.FC<FailedTestGroupProps> = ({
+  tasks,
+  testName,
+}) => {
+  const statusMap = countStatuses(tasks);
+  const sortedStatuses = Object.entries(statusMap).sort((a, b) =>
+    a[0].localeCompare(b[0]),
+  );
+  return (
+    <Accordion
+      shouldRenderChildIfHidden={false}
+      title={
+        <TitleContainer>
+          <Title title={testName}>{trimStringFromMiddle(testName, 120)}</Title>
+          {sortedStatuses.map(([status, count]) => (
+            <TaskStatusBadge
+              key={status}
+              status={status as TaskStatus}
+              taskCount={count}
+            />
+          ))}
+        </TitleContainer>
+      }
+      toggledTitle={
+        <TitleContainer>
+          <Title>{testName}</Title>
+          {sortedStatuses.map(([status, count]) => (
+            <TaskStatusBadge
+              key={status}
+              status={status as TaskStatus}
+              taskCount={count}
+            />
+          ))}
+        </TitleContainer>
+      }
+      useIndent={false}
```

Comment:
We should maybe use indent here, it makes it clear which accordion is open
```suggestion
      useIndent
```